`npm ci` causes failures on VSTS CI under windows hosts.

Travis CI is currently using an updated version of NPM that is calling the new `npm ci` command during our CI build. On VSTS, we aren't able to call this command as it seems to have a problematic effect on the later call to `npm run vscode:prepublish`.

Investigate the problem and fix it.

Error before the fix was:

````
PS D:\dev\github\d3r3kk\vscode-python> npm run vscode:prepublish

> python@2018.6.0-alpha vscode:prepublish D:\dev\github\d3r3kk\vscode-python
> gulp checkNativeDependencies && tsc -p ./

[16:18:54] Using gulpfile D:\dev\github\d3r3kk\vscode-python\gulpfile.js
[16:18:54] Starting 'checkNativeDependencies'...
[16:18:55] 'checkNativeDependencies' errored after 1.31 s
[16:18:55] TypeError: Cannot read property 'toString' of null
    at hasNativeDependencies (D:\dev\github\d3r3kk\vscode-python\gulpfile.js:131:91)
    at Gulp.gulp.task (D:\dev\github\d3r3kk\vscode-python\gulpfile.js:103:9)
    at module.exports (D:\dev\github\d3r3kk\vscode-python\node_modules\orchestrator\lib\runTask.js:34:7)
    at Gulp.Orchestrator._runTask (D:\dev\github\d3r3kk\vscode-python\node_modules\orchestrator\index.js:273:3)
    at Gulp.Orchestrator._runStep (D:\dev\github\d3r3kk\vscode-python\node_modules\orchestrator\index.js:214:10)
    at Gulp.Orchestrator.start (D:\dev\github\d3r3kk\vscode-python\node_modules\orchestrator\index.js:134:8)
    at D:\dev\github\d3r3kk\vscode-python\node_modules\gulp\bin\gulp.js:129:20
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:686:11)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! python@2018.6.0-alpha vscode:prepublish: `gulp checkNativeDependencies && tsc -p ./`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the python@2018.6.0-alpha vscode:prepublish script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
````

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -13,6 +13,7 @@ const tslint = require('tslint');
 const relative = require('relative');
 const ts = require('gulp-typescript');
 const cp = require('child_process');
+const spawn = require('cross-spawn');
 const colors = require('colors/safe');
 const gitmodified = require('gulp-gitmodified');
 const path = require('path');
@@ -128,7 +129,7 @@ function hasNativeDependencies() {
     if (!Array.isArray(nativeDependencies) || nativeDependencies.length === 0) {
         return false;
     }
-    const dependencies = JSON.parse(cp.spawnSync('npm', ['ls', '--json', '--prod']).stdout.toString());
+    const dependencies = JSON.parse(spawn.sync('npm', ['ls', '--json', '--prod']).stdout.toString());
     const jsonProperties = Object.keys(flat.flatten(dependencies));
     nativeDependencies = _.flatMap(nativeDependencies, item => path.dirname(item.substring(item.indexOf('node_modules') + 'node_modules'.length)).split(path.sep))
         .filter(item => item.length > 0)

--- a/package.json
+++ b/package.json
@@ -1954,6 +1954,7 @@
         "chai-as-promised": "^7.1.1",
         "codecov": "^3.0.0",
         "colors": "^1.2.1",
+        "cross-spawn": "^6.0.5",
         "debounce": "^1.1.0",
         "decache": "^4.4.0",
         "del": "^3.0.0",

